python/ray/util/spark/cluster_init.py

```diff
@@ -1231,6 +1303,44 @@ def setup_ray_cluster(
     return cluster.address, remote_connection_address
 
 
+def serve_global_ray_cluster(*args, **kwargs):
```

Comment:
setup_global_ray_cluster

Reply:
If that's the case, the blocking nature should be reflected in the `setup_global_ray_cluster` function name. Otherwise, people will confuse these two.

Maybe this is better: we don't change `setup_ray_cluster` but add a is_blocking param to `setup_global_ray_cluster`.

Reply:
synced offline, still adopt `setup_global_ray_cluster(.., is_blocking)` api